docs/int/quickstart/alone/test-locally.md

```diff
@@ -60,12 +60,10 @@ The default cluster consists of:
    FEE_RECIPIENT_ADDR=<ENTER YOUR FEE RECIPIENT ADDRESS HERE>
 
    # Create a distributed validator cluster
-   docker run --rm -v "$(pwd):/opt/charon" obolnetwork/charon:v0.17.0 create cluster --name="mycluster" --withdrawal-addresses="${WITHDRAWAL_ADDR}" --fee-recipient-addresses="${FEE_RECIPIENT_ADDR}" --nodes 6 --threshold 5 --network goerli --num-validators=1
+   docker run --rm -v "$(pwd):/opt/charon" obolnetwork/charon:v0.17.0 create cluster --name="mycluster" --cluster-dir=".charon/cluster/" --withdrawal-addresses="${WITHDRAWAL_ADDR}" --fee-recipient-addresses="${FEE_RECIPIENT_ADDR}" --nodes 6 --network goerli --num-validators=1
```

Comment:
should it be `--cluster-dir="cluster-nodes"` given the text below?

Reply:
Apologies, walking back from the `cluster-nodes` approach, as it will break existing CDVC deployments which we don't want. Will clear the text too.